src/block-management/block-holder.js

```diff
@@ -20,16 +28,25 @@ import { BlockEdit } from '@wordpress/editor';
 import TextInputState from 'react-native/lib/TextInputState';
 
 type PropsType = BlockType & {
+	clientId: string,
+	rootClientId: string,
 	isSelected: boolean,
+	isFirstBlock: boolean,
+	isLastBlock: boolean,
 	showTitle: boolean,
-	onChange: ( clientId: string, attributes: mixed ) => void,
+	getBlockIndex: ( clientId: string, rootClientId: string ) => number,
+	getPreviousBlockClientId: ( clientId: string ) => string,
+	getNextBlockClientId: ( clientId: string ) => string,
+	onChange: ( attributes: mixed ) => void,
 	onReplace: ( blocks: Array<Object> ) => void,
-	onInlineToolbarButtonPressed: ( button: number, clientId: string ) => void,
-	onSelect: void => void,
-	insertBlocksAfter: ( blocks: Array<Object> ) => void,
-	mergeBlocks: ( forward: boolean ) => void,
-	canMoveUp: boolean,
-	canMoveDown: boolean,
+	onInsertBlocks: ( blocks: Array<Object>, index: number ) => void,
+	onInlineToolbarButtonPressed: ( button: number ) => void,
```

Comment:
Similarly, `onInlineToolbarButtonPressed` is not passed via props anymore, right?